tests/test_api.py

```diff
@@ -1925,7 +1925,7 @@ def test_config_map(self, osbs, name, data):
             'apiVersion': 'v1',
             'kind': 'ConfigMap',
             'metadata': {'name': name},
-            'data': data,
+            'data': {key: json.dumps(value) for key, value in data.items()},
```

Comment:
The test is duplicating functionality. Instead of mocking `create_config_map` it should mock file open and its contents:
```
mock = flexmock(sys.modules['__builtin__'])
mock.should_call('open')  # set the fall-through
(mock.should_receive('open')
    .with_args('/your/file')
    .and_return( flexmock(read=lambda: 'file contents') ))
```